src/Workspaces/Core/Portable/FindSymbols/FindReferences/FindReferencesSearchEngine_DocumentProcessing.cs

```diff
@@ -57,10 +57,11 @@ internal partial class FindReferencesSearchEngine
             {
                 try
                 {
-                    var references = await finder.FindReferencesInDocumentAsync(symbolAndProjectId, document, semanticModel, _cancellationToken).ConfigureAwait(false);
+                    var references = await finder.FindReferencesInDocumentAsync(
+                        symbolAndProjectId, document, semanticModel, _options, _cancellationToken).ConfigureAwait(false);
                     foreach (var location in references)
```

Comment:
>location [](start = 33, length = 8)

nit: consider deconstruction (may be clearer): `foreach (var (location, _) in references) ...`

Reply:
Done.